.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+ - 3.6
+
+addons:
+  apt:
+    packages:
+      - libssl-dev
+      - python3-dev
+
+before_script:
+  - pip install coala-bears[alldeps]==0.11.rc2 --pre -U
```

Comment:
@Mixih stated to use this to get the rc2 in his blog post and gist, so used this. Anyway I'll change this after 0.11 is released